ui/src/app/applications/components/applications-list/applications-tiles.tsx

```diff
@@ -98,6 +98,16 @@ export const ApplicationTiles = ({applications, syncApplication, refreshApplicat
         }
     });
 
+    const getAppName = (appName: string) => {
+        const maxAppNameLength = 30;
+        if (appName.length > maxAppNameLength) {
+            return (
+                <Tooltip content={appName}>
+                    <span className='applications-list__title'>{appName}</span>
+                </Tooltip>
+            );
+        } else return <span className='applications-list__title'>{appName}</span>;
+    };
```

Comment:
As the `if` condition is pretty simple and clear, the return value could be simplified using ternary operator as bellow
```
return appName.length > maxAppNameLength ? 
        (
            <Tooltip content={appName}>
                <span className='applications-list__title'>{appName}</span>
            </Tooltip>
        ) : (<span className='applications-list__title'>{appName}</span>);
```
Check with `make lint-ui` before push though.

Reply:
thanks this looks better!